src/TestLogger/Core/InputSanitizerXml.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Spekt Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Spekt.TestLogger.Core
+{
+    using System.Text.RegularExpressions;
+
+    public class InputSanitizerXml : IInputSanitizer
+    {
+        private static readonly Regex InvalidXmlChar = new (@"([^\x09\x0A\x0D\x20-\uD7FF\uE000-\uFFFD]|[\u007F-\u0084\u0086-\u009F\uFDD0-\uFDEF])", RegexOptions.Compiled);
```

Comment:
One separate note on the regex. I ended up copying in your version of the regex, not the one from trx. They don't exclude the 'discouraged' chars in trx currently. 

The reason I went back to your version is it was needed for the unit tests you had before.

Reply:
Sounds good